tests/pr_test.go

```diff
@@ -119,3 +119,23 @@ func TestRunSecureUpgradeSolution(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestRunBasicExampleWithFlavor(t *testing.T) {
```

Comment:
If this test is just to test passing a value for `member_host_flavor`, can we just use it in one of the other existing tests in this file? ICD had asked us to minimise the number of instances we are creating, and I see no need for a full new test just for this?

Reply:
All changed in latest 3414351. This test now covers the `old` hosting model, whilst it is still supported. The equivalent test in other_test.go is removed. The member_host_flavour is now covered in both complete (as multitenant) and fscloud (as machine type).